krizten/Memoire-WebAPI
# #164633652 Add Entry Endpoint (Memoire REST API)

### What does this PR do?
Include implementation of the add-entry endpoint
#### Description of Task to be completed?
Get the following endpoint working
```POST /api/v1/entries```
#### How should this be manually tested?
After cloning the repo, install the project dependencies and start the project by running ```npm run start:dev```
- Using Postman, test the endpoint to ensure it adds a new entry.
#### What are the relevant pivotal tracker stories?
#164633652
